setup.py

```diff
@@ -1,13 +1,14 @@
 from setuptools import setup, find_packages, Extension
 
 long_description = """\
-This is Python package provides for quick, easy parsing for Cacophony
-Project Thermal Video (CPTV) files. It works with Python 3 only.
+This is Python package provides for quick parsing and generation of
+Cacophony Project Thermal Video (CPTV) files. It works with Python
+3.5+ only.
```

Comment:
is there still f strings? 3.6+?

Reply:
Nope, I got rid of the f-string. We need Python 3.5 support because of the pi.

Reply:
oh ye ofcourse